### PR TITLE
Use inner class as static.

### PR DESCRIPTION
**Pull request by arturobernalg**
Its not require access to an enclosing instance's non-public fields and methods.

### CODE REVIEW

File: src/main/org/apache/tools/ant/taskdefs/Input.java [hunk 1]
```diff
@@ -44,7 +44,7 @@
     /**
      * Represents an InputHandler.
      */
-    public class Handler extends DefBase {
+    public static class Handler extends DefBase {
```
Comment (bodewig) [inline src/main/org/apache/tools/ant/taskdefs/Input.java:47]: this change is not backwards compatible

File: src/main/org/apache/tools/ant/taskdefs/Javac.java [hunk 2]
```diff
@@ -910,7 +910,7 @@ public boolean getNowarn() {
      */
     public ImplementationSpecificArgument createCompilerArg() {
         final ImplementationSpecificArgument arg =
-            new ImplementationSpecificArgument();
+                new ImplementationSpecificArgument();
```
Comment (bodewig) [inline src/main/org/apache/tools/ant/taskdefs/Javac.java:913]: please don't change whitespace in existing code.

**Comment by bodewig** [comment]
Thank you @arturobernalg but I'm afraid this is a change that is not backwards compatible at all so we cannot accept it.

**Comment by arturobernalg** [comment]
> Thank you @arturobernalg but I'm afraid this is a change that is not backwards compatible at all so we cannot accept it.

HI @bodewig 
Ok, close the PR. maybe for a new. mayor release.
TY

**Comment by bodewig** [comment]
for historical reasons Ant is not following semantic versioning, so the next "major release" would be 1.11.0. Having said that I'm not aware of anybody planning to create such a thing.

**Pull request closed without merging**